src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -123,18 +127,37 @@ public boolean link() throws IOException, InterruptedException {
     }
 
     @Override
-    List<String> getTargetSpecificLinkLibraries() {
+    List<String> getTargetSpecificJavaLinkLibraries(){
         List<String> targetLibraries = new ArrayList<>();
 
-        targetLibraries.add("-Wl,-Bstatic");
-        targetLibraries.addAll(asListOfLibraryLinkFlags(staticJavaLibs));
+        Path javaStaticLibPath = null;
+        Path graalClibsPath = getCLibPath();
+        try {
+            javaStaticLibPath = getStaticJDKLibPaths().get(0);
```

Comment:
This should be fixed (in a follow-up change), as there is no need of using a list. For now probably better to check if list is empty before (otherwise the exception won't be catched)